lib/assets/javascripts/turbograft/turbohead.coffee

```diff
@@ -4,39 +4,56 @@ class window.TurboHead
   update: (successCallback, failureCallback) ->
```

Comment:
This function should either:
- Gather a set of `data-turbolinks-track: 'true'` assets, and perform a full page refresh if the list changes
- Fail fast with a, `'true' is no longer supported` exception

The PR in its current form just seems to dynamically load `data-turbolinks-track: true` assets as their fingerprints change.  That's the exact opposite of the old behaviour.